sdks/python/apache_beam/dataframe/frame_base_test.py

```diff
@@ -72,7 +72,7 @@ def add_one(frame):
 
   def test_args_to_kwargs(self):
     class Base(object):
-      def func(self, a=1, b=2, c=3):
+      def func(self, a=1, b=2, c=3, *, kw_only=4):
```

Comment:
would it make sense to also include an argument without a default in this or antoher test case?

Reply:
Done